mParticle-Leanplum/MPKitLeanplum.m

```diff
@@ -43,7 +43,8 @@ - (nonnull instancetype)initWithConfiguration:(nonnull NSDictionary *)configurat
     NSString *appId = configuration[@"appId"];
     NSString *clientKey = configuration[@"clientKey"];
     NSString *userIdField = configuration[@"userIdField"];
-    if (!self || !appId || !clientKey || !userIdField) {
+    NSString *deviceIdField = configuration[@"iosDeviceId"];
```

Comment:
Is this a required or optional value? If optional the following `if` could fail and return `nil`